localstack/services/events/provider_v2.py

```diff
@@ -79,6 +85,39 @@ def get_filtered_dict(name_prefix: str, input_dict: dict) -> dict:
     return {name: value for name, value in input_dict.items() if name.startswith(name_prefix)}
 
 
+def get_event_time(event: PutEventsRequestEntry) -> str:
```

Comment:
style: the `event` is not needed for a time conversion. Passing minimal knowledge is considered best practice. Logging could still happen at a different place.

Reply:
true, but in this case the function not only serves to convert time but also conditionally set the time based if the time filed is set in events or not.

Reply:
ok, then the naming `get_event_time` is a bit misleading here. Probably worth testing/refactoring in a follow-up.